exporters/variables-scss/src/content/formatters/__tests__/cssFormatter.test.ts

```diff
@@ -0,0 +1,16 @@
+import fs from 'fs';
+import path from 'path';
+import { formatCSS } from '../cssFormatter';
+
+const mockedUnformattedCSS = fs.readFileSync(
+  path.join(__dirname, '../../__fixtures__/unformattedExample.scss'),
```

Comment:
```suggestion
  path.join(__dirname, '/__fixtures__/unformattedExample.scss'),
```

Reply:
Please, move `__fixtures__` directory close to the tests, e.g. `__tests__/__fixtures__`.